linkedList.js

```diff
@@ -4,5 +4,110 @@
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
 function linkedListGenerator(){
+  var head = null;
+  var tail = null;
+  var node = function() {
+    this.value = null;
+    this.next = null;
+  };
 
+ function getHead(){
+  return head;
+ }
+ function getTail(){
+  return tail;
+ }
+
+ function add(item){
+  var insert = new node();
+  insert.value = item;
+    if(head === null){
+      head = insert;
+      head.next = tail;
```

Comment:
If the new node becomes the head, then next is already set to `null`. It doesn't really make sense to say that `head.next = tail` because this is not true, since the head *is* the tail. I think makes more sense to set it explicitly to `null`, but that is already done on line 11.

I think this line can be safely removed.